apps/web/app/hooks/features/useKanban.ts

```diff
@@ -73,14 +71,80 @@ export function useKanban() {
 			});
 	};
 
+	const addNewTask = (task: ITeamTask, status: string) => {
```

Comment:
@Anishali2 I am not sure what it's doing here with all that hardcoded test data etc :)

Reply:
This is created for testing purpose
Once i go through this project so i can use that in kanban section in future i can remove that i and make a backup my self